app/src/main/java/edu/uc/app/parkfinder/MainActivity.kt

```diff
@@ -11,8 +11,8 @@ class MainActivity : AppCompatActivity() {
         setContentView(R.layout.main_activity)
```

Comment:
All updates to this file coding styles according to: https://developer.android.com/kotlin/style-guide